tests/receivers/oracledb/testdata/resource_metrics/all.yaml

```diff
@@ -1,8 +1,7 @@
 resource_metrics:
-  - attributes:
-      oracledb.instance.name: localhost:1521/XE
```

Comment:
did you find this necessary?

Reply:
sorry I misunderstood. Do you mean is this necessary?
```
- instrumentation_scope:
          attributes: {}
      ```
      
     Or do you mean "do you need this? to be here?"
     ```
     - attributes:
      oracledb.instance.name: localhost:1521/XE
      ```

Reply:
For Context:
```
removing oracledb.instance.name: localhost:1521/XE since it was already passing and your changes are additive in another section

I think when you add attributes: {} you're enforcing that the attributes are empty where omitting them means they're unvetted

or at least I think that's my expected behavior but I'm not sure it works in the scope
```

Ended up needing some additional tweaks to get this to work and some checking of other fields like `container.name` and `container.image.name` (which is generated so match `<ANY>`)